main.gms

```diff
@@ -230,548 +217,1220 @@ $setGlobal c_description  REMIND run with default settings
+;
+  cm_noPeFosCCDeu = 0; !! def 0
+parameter
+  cm_logitCal_markup_conv_b   "value to which logit calibration markup of standard fe2ue technologies in detailed buildings module converges to"
+*** def 0.8, long-term convergence value of detailed buildings fe2ue conventional techs price markup
+;
+  cm_logitCal_markup_conv_b = 0.8; !! def 0.8
+parameter
+  cm_logitCal_markup_newtech_conv_b "value to which logit calibration markup of new fe2ue technologies in detailed buildings module converges to"
+***  def 0.3, long-term convergence value of detailed buildings fe2ue new techs price markup
+;
+  cm_logitCal_markup_newtech_conv_b = 0.3; !! def 0.3
+parameter
   cm_startIter_EDGET          "starting iteration of EDGE-T"
-  c_VREPot_Factor             "switch for rescaling renewable potentials in all grades which have not been used by 2020"
-  cm_FEtax_trajectory_abs     "switch for setting the aboslute FE tax level explicitly from a given year onwards, before tax levels increases or decreases linearly to that value"
-  cm_FEtax_trajectory_rel     "factor for scaling the FE tax level relative to cm_startyear from a given year onwards, before tax levels increases or decreases linearly to that value"
-  cm_CESMkup_ind                 "switch for setting markup cost to CES nodes in industry" 
-  cm_CESMkup_build               "switch for setting markup cost to CES nodes in buildings" 
+*** EDGE-T transport starting iteration of coupling
+*** def 14, EDGE-T coupling starts at 14, if you want to test whether infeasibilities after EDGE-T -> set it to 1 to check after first iteration
+;
+  cm_startIter_EDGET = 14; !! def 14, by default EDGE-T is run first in iteration 14
+parameter
   c_BaselineAgriEmiRed     "switch to lower agricultural base line emissions as fraction of standard assumption, a value of 0.25 will lower emissions by a fourth"
+*** switch to lower Baseline agriultural emissions in all regions, value is the fraction of reduction with respect to default assumptions,
+*** e.g. 0.4 means 40% lower emissions trajectory relative to default, reduction starts in 2030, reaches full reduction by 2040
+;
+  c_BaselineAgriEmiRed = 0; !! def = 0
+parameter
   cm_deuCDRmax                 "switch to limit maximum annual CDR amount in Germany in MtCO2 per y"
-  cm_EnSecScen             "switch for running an ARIADNE energy security scenario, introducing a tax on PE fossil energy in Germany"
-  cm_Ger_Pol               "switch for selecting different policies for Germany used in the ARIADNE scenarios"
-  cm_process_based_steel      "switch to turn on process-based steel implementation"
+*** switch to cap annual DEU CDR amount by value assigned to switch, or no cap if -1, in MtCO2
 ;
-
-*** --------------------------------------------------------------------------------------------------------------------------------------------------------------------
-***                           YOU ARE IN THE WARNING ZONE (DON'T DO CHANGES HERE)
```

Comment:
This has to be removed, right?

Reply:
it is isn't it?

Reply:
sorry, I meant it has to be removed also from other places where it is still there

Reply:
I cannot find it anywhere in main.gms